CHANGELOG.md

```diff
@@ -1,50 +1,71 @@
-- Merge pull request #57 from fernandopradocabrillo/update-tef-codeowners
-- update codeowners from Telefónica
-- Merge pull request #56 from camaraproject/bigludo7-patch-1
-- Update number_verification.yaml
-- Merge pull request #53 from DT-DawidWroblewski/main
-- update to yaml
-- Merge branch 'main' of https://github.com/DT-DawidWroblewski/NumberVerification
-- adding fixes to PR
-- Merge pull request #52 from jlurien/chore/replace-polymorphism-requestBody
-- Replace polymorphism in requestBody of POST /verify
-- Update documentation/API_documentation/README.MD
-- Merge pull request #50 from monamok/chore/update_documentation
-- cleaning repo
-- Relevant definition section added
-- Merge pull request #43 from monamok/chore/phone_number_description_alignment
-- error code copy/pase typo removed
-- updating uml url
-- phoneNumber descriptions aligned to the other APIs
+* User Story in documentation/API_documentation directory by @bigludo7 [PR118](https://github.com/camaraproject/NumberVerification/pull/118)
+* Test Definition in Test_Definitions directory [To be done]
 
 ### Changed
 
-* N/A
+* Aligned with CAMARA design guidelines & Identity Consent management
+* Make the '+' mandatory for the phone number by @fernandopradocabrillo [PR90](https://github.com/camaraproject/NumberVerification/pull/90)
+* Cosmetic change following megalinter integration by @bigludo7 [PR103](https://github.com/camaraproject/NumberVerification/pull/103)
+* Update Authorization and authentication part accordingly to ICM by @fernandopradocabrillo [PR88](https://github.com/camaraproject/NumberVerification/issues/88)
+* Adding a pattern for PhoneNumber in /verify by @maxl2287 [PR68](https://github.com/camaraproject/NumberVerification/issues/76)
+* Clarify use of 'user' and 'subscriber' wording by @AxelNennker [PR102](https://github.com/camaraproject/NumberVerification/pull/102)
 
 ### Fixed
 
-* N/A
+* Replaced OAuth2 auth code flow by OIDC auth code flow by @AxelNennker [PR109](https://github.com/camaraproject/NumberVerification/pull/109)
 
 ### Removed
 
-* N/A
\ No newline at end of file
+* n/a
+
+## New Contributors 
+
+- @AxelNennker made their first contribution in clarifying use of 'user' and 'subscriber' wording by @AxelNennker [PR102](https://github.com/camaraproject/NumberVerification/pull/102)
+- @rartych made their first contribution in GitHub workflows [#108](https://github.com/camaraproject/NumberVerification/pull/108)
+- @fernandopradocabrillo made their first contribution by updating Authorization and authentication part accordingly to ICM [PR88](https://github.com/camaraproject/NumberVerification/issues/88)
+- @maxl2287 made their first contribution by adding a pattern for PhoneNumber in /verify  [PR68](https://github.com/camaraproject/NumberVerification/issues/76)
+- @bigludo7 made their first contribution by making change following megalinter integration [PR103](https://github.com/camaraproject/NumberVerification/pull/103)
+
+
```

Comment:
My attempt to add v0.3.1 changes (derived from https://github.com/camaraproject/NumberVerification/releases/tag/v0.3.1, shortened):

```suggestion
**Full Changelog**: https://github.com/camaraproject/NumberVerification/compare/v0.3.1...r1.1

## v0.3.1

Initital release of Camara Number Verification API

## What's Changed
* Telefonica Proposal by @monamok in https://github.com/camaraproject/NumberVerification/pull/3
* Adding API documentation by @monamok in https://github.com/camaraproject/NumberVerification/pull/6
* Initial content for Number Verify by @DT-DawidWroblewski in https://github.com/camaraproject/NumberVerification/pull/2
* New specific 403 token error and guidelines alignment by @monamok in https://github.com/camaraproject/NumberVerification/pull/19
* adding puml by @DT-DawidWroblewski in https://github.com/camaraproject/NumberVerification/pull/24
* guidelines alignment errors and camel case by @monamok in https://github.com/camaraproject/NumberVerification/pull/30
* Change 'sub' cardinality + add attribute description by @bigludo7 in https://github.com/camaraproject/NumberVerification/pull/32
* Embed documentation in API Spec by @monamok in https://github.com/camaraproject/NumberVerification/pull/38

**Full Changelog**: https://github.com/camaraproject/NumberVerification/commits/v0.3.1
```